workers/unity/Assets/Tests/EditmodeTests/Correctness/PlayerLifecycle/PlayerLifecycleHelperTests.cs

```diff
@@ -16,6 +15,7 @@ public class PlayerLifecycleHelperTests
         [SetUp]
         public void Setup()
         {
+            // TODO: The WorkerEntityId for this worker is '0'. Probably should refactor this in the future.
```

Comment:
Worth doing now, or is this a future thing?

Reply:
Not impacting anything now, so no need right now. If we run into a case where we need to do it, we can tweak it